service/storage_service.cc

```diff
@@ -449,7 +449,6 @@ future<> storage_service::sync_raft_topology_nodes(mutable_token_metadata_ptr tm
             sys_ks_futures.push_back(_sys_ks.local().update_peer_info(*ip, host_id, info));
             if (!prev_normal.contains(id)) {
                 co_await notify_joined(*ip);
-                co_await remove_rpc_client_with_ignored_topology(*ip);
```

Comment:
IIUC we could probably keep this, it shouldn't hurt.

Reply:
why?

Reply:
why it shouldn't hurt -- it should be no-op
why keep it -- just in case? Dunno, probably unnecessary.

Reply:
No need to keep it then since nobody will remember that it is unnecessary and somebody will spend time figuring out why it is here.